flare/framework/env.py

```diff
@@ -1,25 +1,163 @@
 import gym
 import numpy as np
 from scipy import misc
+from abc import ABCMeta, abstractmethod
+from py_simulator import Simulator
```

Comment:
This might cause errors if the user hasn't installed py_simulator yet. Probably we should put the import inside the class?